DWAPlannerROS:
  odom_topic: odom
  xy_goal_tolerance: 0.2 # [m]
  transform_tolerance: 0.5 # [s]

  # Robot
  max_vel_x: 1 # [m/s]
  min_vel_x: -1 # [m/s]
  max_vel_theta: 1.2 # [rad/s]
  min_vel_theta: -1.2 # [rad/s]
  acc_lim_x: 0.5 # [m/s^2]
  acc_lim_theta: 1.2 # [rad/s^2]
  control_period: 0.2 # [s]

  # DWA
  sim_time_samples: 10
  vx_samples: 10
  vth_samples: 20
  path_distance_bias: 32.0
  goal_distance_bias: 24.0
  occdist_scale: 0.01
